.eslintignore

```diff
@@ -1,4 +1,16 @@
-bin
-packages
-templates
+# don't lint the linter config
+.eslintrc.js
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
 dist
+build
+# don't lint nyc coverage output
+coverage
+# don't lint generated types
+**/generated/types.d.ts
+**/generated/types.ts
+# don't lint ide files
+.idea
+.vscode
+src/templates
```

Comment:
What's this?

Reply:
Ahhh I see

Reply:
I think it's not needed - originally the eslint script was using `src/**`, changing to `src/**/*.ts` fixed the issue I was having (see previous build failures)